### PR TITLE
130 int availability

### PR DESCRIPTION
**Pull request by michel-fresh**
https://justice.gov.bc.ca/jira/browse/CSBISS-130

### CODE REVIEW

**Review by jerryfreshworks — APPROVED** [review]
Perfect!
Let's wait until openshift is up then merge?
or if you have another new PR working on, then welcome to merge this.

**Comment by michel-fresh** [comment]
@jerryfreshworks I don't but I think OCP is back up so let's see

**Pull request merged**
